clang/test/AST/ByteCode/amdgpu-nullptr.cl

```diff
@@ -0,0 +1,12 @@
+// RUN: %clang_cc1 -no-enable-noundef-analysis %s -cl-std=CL2.0 -include opencl-c.h -triple amdgcn -emit-llvm -o - | FileCheck %s
```

Comment:
I don't think you need the `-include opencl-c.h` for this test? (it's rather big, so slows down testing)

Same comment applies to the next RUN line.

Reply:
Ah, nice. Thanks